### ISSUE TITLE
Invalid Timezone Exception In Tests and Prototype

### ISSUE THREAD

**Issue by cmuell89** [FraBle/python-duckling#8]
Running your tests I get the following error:

```
(env) cmuell89 [~/git/python-duckling] $ python -m pytest duckling/
=============================================================================================== test session starts ===============================================================================================
platform linux -- Python 3.5.2+, pytest-3.0.6, py-1.4.32, pluggy-0.4.0
rootdir: /home/cmuell89/git/python-duckling, inifile: 
plugins: cov-2.4.0
collected 65 items 

duckling/test/test_duckling.py ..........................
duckling/test/test_duckling_wrapper.py ....FF...........................FF....

==================================================================================================== FAILURES =====================================================================================================
________________________________________________________________________________ test_parse_time_with_reference_time_and_timezone _________________________________________________________________________________

duckling_wrapper = <duckling.wrapper.DucklingWrapper object at 0x7fbff39ed198>

    def test_parse_time_with_reference_time_and_timezone(duckling_wrapper):
        result = duckling_wrapper.parse_time(
>           u'Let\'s meet tomorrow at 12pm', reference_time=u'1990-12-30 15:30:00-8:00')

duckling/test/test_duckling_wrapper.py:63: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
duckling/wrapper.py:263: in parse_time
    reference_time=reference_time)
duckling/wrapper.py:56: in _parse
    reference_time=reference_time)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <duckling.duckling.Duckling object at 0x7fbff39ed7b8>, input_str = "Let's meet tomorrow at 12pm", language = 'en$core', dim_filter = 'time', reference_time = '1990-12-30 15:30:00-8:00'

    def parse(self, input_str, language=Language.ENGLISH, dim_filter=None, reference_time=''):
        """Parses datetime information out of string input.
    
            It invokes the Duckling.parse() function in Clojure.
            A language can be specified, default is English.
    
            Args:
                input_str: The input as string that has to be parsed.
                language: Optional parameter to specify language,
                    e.g. Duckling.ENGLISH.
                dim_filter: Optional parameter to specify list of filters for
                    dimensions in Duckling.
                reference_time: Optional reference time for Duckling.
    
            Returns:
                A list of dicts with the result from the Duckling.parse() call.
    
            Raises:
                RuntimeError: An error occurres when Duckling model is not loaded
                    via load().
            """
        if self._is_loaded is False:
            raise RuntimeError(
                'Please load the model first by calling load()')
        duckling_parse = self.clojure.var("duckling.core", "parse")
        duckling_time = self.clojure.var("duckling.time.obj", "t")
        clojure_hashmap = self.clojure.var("clojure.core", "hash-map")
    
        filter_str = '[]'
        if dim_filter:
            filter_str = '[:{filter}]'.format(filter=dim_filter)
    
        if reference_time:
            duckling_result = duckling_parse.invoke(
                language,
                input_str,
                self.clojure.read(filter_str),
                clojure_hashmap.invoke(
                    self.clojure.read(':reference-time'),
                    duckling_time.invoke(
>                       *self._parse_reference_time(reference_time))
                )
            )
E           jpype._jexception.clojure.lang.ExceptionInfoPyRaisable: clojure.lang.ExceptionInfo: Invalid timezone {:tz 8.0}

duckling/duckling.py:127: clojure.lang.ExceptionInfoPyRaisable

```

I also get the same error attempting to pass your test reference time to my prototyping file for the parse method of the lower level duckling class.

**Comment by FraBle** [comment]
That's very weird. The tests are running fine for me on OSX with Python2 and Python3. Have you been able to solve it?
I'm currently not able to test it further right now (I'm off for my wedding).
Starting from March I should be back on track.

**Comment by FraBle** [comment]
@cmuell89, updates on reproducing it?

**Comment by FraBle** [comment]
Fixed and released with version [v1.3.2](https://pypi.python.org/pypi?:action=display&name=duckling&version=1.3.2) :)
